src/Template/Layout/js/app/app.js

```diff
@@ -374,25 +376,110 @@ const _vueInstance = new Vue({
          *
          * @returns {void}
          */
-        alertBeforePageUnload() {
-            var forms = [...document.querySelectorAll('form')];
-            forms.forEach((form) => {
-                form.addEventListener('change', (ev) => {
-                    form.changed = true;
-                });
-                form.addEventListener('submit', (ev) => {
+        alertBeforePageUnload(view) {
```

Comment:
do we need this change here? or is it a general optimization?

Probably a PR with new `alert before unload` logic is preferrable, it seems a significant change...